ShapeEditorWindow.cs error

Hi.

I put the SabreCSG-master zip file content into assets folder and i am getting following errors. There is no sabrecsg object in the GameObject menu. I tested with Unity 5.6.5f1 and 2017.1.2f1.

Assets/SabreCSG-master/Scripts/Brushes/CompoundBrushes/Editor/ShapeEditorWindow.cs(990,49): error CS0117: `UnityEngine.Screen' does not contain a definition for `safeArea'

Assets/SabreCSG-master/Scripts/Brushes/CompoundBrushes/Editor/ShapeEditorWindow.cs(990,113): error CS0117: `UnityEngine.Screen' does not contain a definition for `safeArea'

Assets/SabreCSG-master/Scripts/Brushes/CompoundBrushes/Editor/ShapeEditorWindow.cs(1350,40): error CS0117: `UnityEngine.Screen' does not contain a definition for `safeArea'

Assets/SabreCSG-master/Scripts/Brushes/CompoundBrushes/Editor/ShapeEditorWindow.cs(1480,47): error CS0117: `UnityEngine.Screen' does not contain a definition for `safeArea'

Assets/SabreCSG-master/Scripts/Brushes/CompoundBrushes/Editor/ShapeEditorWindow.cs(1480,131): error CS0117: `UnityEngine.Screen' does not contain a definition for `safeArea'

--- a/Scripts/Brushes/CompoundBrushes/Editor/ShapeEditorWindow.cs
+++ b/Scripts/Brushes/CompoundBrushes/Editor/ShapeEditorWindow.cs
@@ -986,7 +986,7 @@ namespace Sabresaurus.SabreCSG.ShapeEditor
         private void OnHome()
         {
             // scroll to the center of the screen.
-#if UNITY_5_4_OR_NEWER
+#if UNITY_2017_2_OR_NEWER
             viewportScroll = new Vector2(Screen.safeArea.width / 2.0f / EditorGUIUtility.pixelsPerPoint, Screen.safeArea.height / 2.0f / EditorGUIUtility.pixelsPerPoint);
 #else
             viewportScroll = new Vector2(Screen.width / 2.0f, Screen.height / 2.0f);
@@ -1346,7 +1346,7 @@ namespace Sabresaurus.SabreCSG.ShapeEditor
 
         private Rect GetViewportRect()
         {
-#if UNITY_5_4_OR_NEWER
+#if UNITY_2017_2_OR_NEWER
             Rect viewportRect = Screen.safeArea;
 #else
             Rect viewportRect = new Rect(0, 0, Screen.width, Screen.height);
@@ -1476,7 +1476,7 @@ namespace Sabresaurus.SabreCSG.ShapeEditor
         private void ShowCenteredPopupWindowContent(PopupWindowContent popup)
         {
             Vector2 size = popup.GetWindowSize();
-#if UNITY_5_4_OR_NEWER
+#if UNITY_2017_2_OR_NEWER
             PopupWindow.Show(new Rect((Screen.safeArea.width / 2.0f / EditorGUIUtility.pixelsPerPoint) - (size.x / 2.0f), (Screen.safeArea.height / 2.0f / EditorGUIUtility.pixelsPerPoint) - (size.y / 2.0f), 0, 0), popup);
 #else
             PopupWindow.Show(new Rect((Screen.width / 2.0f) - (size.x / 2.0f), (Screen.height / 2.0f) - (size.y / 2.0f), 0, 0), popup);

